src/components/WarningCard/styles.ts

```diff
@@ -29,14 +28,20 @@ export const WarningCard = styled.div<IWarningCardProps>`
     transition-property: opacity;
 
     img {
-      margin-top: 0.1rem;
+      display: none;
     }
   `}
 
   ${({ showCard }) =>
```

Comment:
In my opinion, there is no need to create another arrow function to destructure `showCard`, but I guess it's just an opinion